backend/app/graphql/meal_request.py

```diff
@@ -47,7 +53,7 @@ class Arguments:
         meal_info = MealTypeInput(required=True)
         drop_off_time = graphene.Time(required=True)
         drop_off_location = graphene.String(required=True)
-        delivery_instructions = graphene.String()
+        delivery_instructions = graphene.String(default_value=None)
```

Comment:
Without specifying the `default_value`, the mutation would not allow for this field to be left empty.